common/contracts/assemblyscript/compile.js

```diff
@@ -43,8 +43,10 @@ if (code === 0 && calledFromDir !== __dirname) {
   const outFile = `./build/${debug ? 'debug' : 'release'}/${packageName}.wasm`
   sh.mkdir('-p', linkDir)
   sh.rm('-f', link)
-  const linkPath = path.relative(linkDir, outFile)
-  sh.ln('-s', linkPath, link)
+  //fix #831: copy-update instead of linking .- sometimes sh.ln does not work on Windows
+  //const linkPath = path.relative(linkDir, outFile)
+  //sh.ln('-s', linkPath, link)
```

Comment:
Can get rid of these comments

Reply:
leave this 

`//fix #831: use copy-update instead of linking .- sometimes sh.ln does not work on Windows`

so this is not reintroduced in the future

Reply:
#fix,  #fixes, #fixed... all link an Issue to a PR